zcash_client_backend/src/data_api/wallet.rs

```diff
@@ -4,7 +4,7 @@ use std::fmt::Debug;
 use zcash_primitives::{
     consensus::{self, BranchId, NetworkUpgrade},
     memo::MemoBytes,
-    prover::TxProver,
+    sapling::prover::TxProver,
```

Comment:
I don't entirely agree with this refactor; `TxProver` was intended to be a generic prover interface that handled whatever proofs were necessary. But currently in the `orchard` crate we are inlining proof creation, so let's do this refactor for now and then figure out a common prover interface later.